pkg/ddevapp/ddevapp_test.go

```diff
@@ -235,6 +235,20 @@ var (
 			DynamicURI:                    testcommon.URIWithExpect{URI: "/", Expect: "This is a test page"},
 			FilesImageURI:                 "/sites/default/files/2022-07/Logo.png",
 		},
+		// 13: craftcms
+		{
+			Name:                          "TestPkgCraftCms",
+			SourceURL:                     "https://craftcms.com/latest-v4.zip",
+			ArchiveInternalExtractionPath: "craftcms-v4",
+			FilesTarballURL:               "",
+			DBTarURL:                      "",
```

Comment:
DBTarURL is a tarball containing db.sql with your database. It's lost to history why this is a tarball.

Reply:
Ok, well here it is: [db-v4.2.3.sql.tar.gz](https://github.com/khalwat/ddev/files/9485454/db-v4.2.3.sql.tar.gz)